### PR TITLE
Implement reducer tests for `events` #397

### PR DESCRIPTION
**Pull request by Mouzourides**
### Implement reducer tests for `events` 

#397 

### What's Changed

Test reducer for events

### Technical Description

Implement Jest tests for the events reducer by testing state when the three diferent actions are called.

### CODE REVIEW

File: services/client/src/pages/events/action-creators/tests/index.spec.js [hunk 1]
```diff
@@ -79,3 +80,66 @@ describe("actions", () => {
...
+      {
+        ...defaultState,
+        hasErrors: true,
+        isLoading: false
+      }
+    );
+  });
+
+  it("should return the state with updated events", () => {
+    const event = {
```
Comment (apoclyps) [inline services/client/src/pages/events/action-creators/tests/index.spec.js]: There are a few additional fields stored in state from the [`events` endpoint](https://muxer.co.uk/events); 
```
{
category: "Technology Monthly"
created: "2018-07-29T13:52:46.642580"
deleted: null
description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
duration: 10000
end: "2018-10-03T19:00:00"
entry: [{description: "", id: "ea7c5b64-50be-439d-900a-6ed60fcf1ea3", type: "free"}]
id: "0611f963-0f2f-4bd3-8dc4-b3dea517f16f"
meetup: []
name: "Monthly Meetup"
source: "meetup"
start: "2018-10-03T19:00:00"
topics: ["technology"]
updated: "2018-07-29T13:52:46.642589"
url: "https://www.example.com/events/253133796/"
}
```

File: services/client/src/pages/events/action-creators/tests/index.spec.js [hunk 2]
```diff
@@ -79,3 +80,79 @@ describe("actions", () => {
...
+      name: "Monthly Meetup",
+      source: "meetup",
+      start: "2018-10-03T19:00:00",
+      topics: ["technology"],
+      updated: "2018-07-29T13:52:46.642589",
+      url: "https://www.example.com/events/253133796/"
+    };
+    const updatedEvent = {
+      ...event,
+      timestamp: 1538589600000
```
Comment (apoclyps) [inline services/client/src/pages/events/action-creators/tests/index.spec.js]: Tests are failing on an incorrect timestamp: you should be able to see the failing test run [here](https://travis-ci.com/apoclyps/my-dev-space/builds/86801884#L2926)

There is a mismatch between expected and actual

```
    -      "timestamp": 1538589600000,
    +      "timestamp": 1538593200000,
```

Comment (Mouzourides) [reply services/client/src/pages/events/action-creators/tests/index.spec.js]: Weird, it passes locally. I'll just use moment to calculate the timestamp then

**Comment by apoclyps** [comment]
Thank you @Mouzourides for submitting this PR

LGTM 👍

**Pull request merged**